Categories/middleware/verify_jwt.js

```diff
@@ -0,0 +1,19 @@
+import jwt from 'jsonwebtoken';
+const TOKEN_KEY = '' + process.env.SECRET_KEY;
+
+const authenticateToken = async (req, res, next) => {
+  if (req.headers.authorization) {
+    const token = req.headers.authorization.split('Bearer ')[1];
+    jwt.verify(token, TOKEN_KEY, (err) => {
+      if (err) {
+        res.status(401).json({ error: 'AUTH_ERROR' });
+      } else {
+        next();
+      }
+    });
+  } else {
+    res.status(401).json({ error: 'AUTH_ERROR_FROM_AUTHCHECKER' });
+  }
+};
```

Comment:
이건 언제 발생하는 에러인가요?
코드로 보았을 때는 토큰이 없을 때 일어나는 에러인 것 같습니다 :)
제 생각이 맞을까요?
만약 맞다면, 토큰이 필요하다나는 사실을 프론트에게 알려주는 것이 좋습니다.
에러 메세지가 모호한데, 이렇게 두루뭉술한 에러 메세지는 회원가입/로그인 때만 해주시는 것이 좋고,
그 외의 상황에는 정확하게 어떤 상황인데 프론트에게 알려주는 쪽을 추천드립니다 👍 
```suggestion
    });
  } else {
    res.status(401).json({ error: 'USER_TOKEN_REQUIRED' });
  }
};
```

Reply:
네! 두루뭉술한 에러 메세지가 아닌 명확한 메세지를 전달할 수 있도록 수정해보겠습니다. 감사합니다 :)